tutorials/src/bin/basic-tutorial-6.rs

```diff
@@ -122,39 +122,36 @@ fn main() {
     loop {
         let msg = bus.timed_pop(gst::CLOCK_TIME_NONE);
 
-        match msg {
-            Some(msg) => {
-                match msg.view() {
-                    MessageView::Error(err) => {
+        if let Some(msg) = msg {
```

Comment:
You can even get rid of the loop here.
```rust
while let Some(msg) = bus.timed_pop(...) {
  [...]
}
```

Reply:
True!

Didn't know about while let. Cool.

Will update.